cita-auth/src/block_verify.rs

```diff
@@ -71,6 +75,23 @@ impl<'a> BlockVerify<'a> {
         true
     }
 
+    fn verify_min_quota(&self, sig_tx: &SignedTransaction) -> bool {
```

Comment:
should add a comment for this method

Reply:
If you follow the way of Ethereum, then:
```
21000 + (68 or 4 per byte) + (32000 if tx.to == 0)
```

create_contract: 53000

Reply:
I will change amend it.